controllers/rcj_soccer_team_blue/rcj_soccer_robot.py

```diff
@@ -145,7 +154,7 @@ def is_new_ball_data(self) -> bool:
         """
         return self.ball_receiver.getQueueLength() > 0
 
-    def get_gps_coordinates(self):
+    def get_gps_coordinates(self) -> list:
```

Comment:
@Adman just out of curiosity, doesn't this require Python version of at least 3.9 (https://www.python.org/dev/peps/pep-0585/)? Should we then perhaps think about supporting just 3.9+ instead of 3.8+ that seems to be mentioned in the docs now?

Reply:
@mrshu I think `list` is OK in py3.8. In py3.9 it is possible to do `list[float]`, while in py3.8 it is only possible to do `List[float]` where `List` is imported from `typing` library.

Reply:
You are indeed right @Adman, this should have no bearing (even within `mypy`).